man/systemd.unit.xml

```diff
@@ -467,10 +467,12 @@
     <programlisting>systemd-analyze --user unit-paths</programlisting>
     </para>
 
-    <para>Moreover, additional units might be loaded into systemd ("linked") from
-    directories not on the unit load path. See the <command>link</command> command
-    for
-    <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+    <para>Moreover, additional units might be loaded into systemd from
+    directories not on the unit load path by creating a symlink pointing to a
+    unit file in the directories. You can use <command>systemctl list</command>
```

Comment:
I don't think systemctl list exists

Reply:
Thanks. I didn't realized that at all. I meant "link" of course. I've already force pushed the fix.